x-pack/plugins/fleet/server/services/agent_policies/package_policies_to_agent_inputs.ts

```diff
@@ -152,10 +161,26 @@ export const storedPackagePoliciesToAgentInputs = async (
         packagePolicy,
         packageInfo,
         outputId,
-        agentPolicyNamespace
+        agentPolicyNamespace,
+        addFields
       )
     );
   }
 
   return fullInputs;
 };
+
+const globalDataTagsToAddFields = (tags: GlobalDataTag[]): FullAgentPolicyAddFields => {
```

Comment:
can we cover with unit/integration tests?

Reply:
I checked the tests for this file, there only seems to be a test suite for [storedPackagePoliciesToAgentInputs](https://github.com/elastic/kibana/blob/main/x-pack/plugins/fleet/server/services/agent_policies/package_policies_to_agent_inputs.test.ts#L32). Any functionality in [storedPackagePolicyToAgentInputs](https://github.com/elastic/kibana/blob/main/x-pack/plugins/fleet/server/services/agent_policies/package_policies_to_agent_inputs.ts#L25) seems to be tested through `storedPackagePoliciesToAgentInputs`. So right now I'm in between creating a new test suite for `storedPackagePolicyToAgentInputs` or keeping everything in `storedPackagePoliciesToAgentInputs`. Regardless of how I go about doing it, I'm planning on testing `globalDataTagsToAddFields` through either one of these functions.

Reply:
we also have API integration tests in `fleet_api_integration` folder that should cover this API